wiki/Getting_Started_Contributing_to_Wiki/en.md

```diff
@@ -0,0 +1,30 @@
+Getting Started on contributing to the osu!Wiki
+==================
+
+Thanks for your interest in making osu!Wiki Better! As you can see, we have moved everything to [GitHub](https://github.com/ppy/osu-wiki).
+So here's some few things in mind to get you started in Contributing to the new Wiki.
+
+
+### Prerequisites
+
+To be able to contribute to osu!Wiki you must have the following :
+
+- Basic Knowledge in VCS (Version Control System), Particularly Git (If you plan to make changes locally).
+- Basic Knowledge in Markdown.
+- a Github Account.
+- a Markdown Editor. We either use [Visual Studio Code](https://code.visualstudio.com) or [Atom](https://atom.io) but in some cases, Github's web Editor will suffice.
+- a Markdown Previewer with **GFM** Support (Github-Flavoured Markdown). We use [Shiba](https://github.com/rhysd/Shiba) or Atom and Visual Studio Code's Built In Markdown Previewers (If you plan to make changes locally).
```

Comment:
Mark this and the one before this as optional. While they're good things to have, we shouldn't impose them to use them.